mmocr/core/visualize.py

```diff
@@ -776,6 +781,14 @@ def draw_edge_result(img, result, edge_thresh=0.5, keynode_thresh=0.5):
                 pred_edge_img, (pos_right_bottom[0] + 5, bbox_y1 + 10),
                 (bbox_x1 + dist_key_to_value - 5, bbox_y1 + 10), arrow_color,
                 1)
+            score_font_size = int(key_font_size * 0.3)
+            score_pos_x = int(
+                (pos_right_bottom[0] + bbox_x1 + dist_key_to_value) / 2.)
+            score_pos_y = bbox_y1 + 10 - score_font_size
+            score_color = (0, 255, 0)
```

Comment:
Reorganize code to avoid some duplicated lines:
Move `score_color...` to line 744
Place `cv2.putText...` right after this if-else block